hooks/tk-multi-publish2/basic/nuke_publish_reviewsubmission.py

```diff
@@ -0,0 +1,333 @@
+﻿# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import nuke
+import sgtk
+
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class NukeReviewSubmissionPublishPlugin(HookBaseClass):
+    """
+    Plugin for publishing an open nuke session.
+
+    This hook relies on functionality found in the base file publisher hook in
+    the publish2 app and should inherit from it in the configuration. The hook
+    setting for this plugin should look something like this::
+
+        hook: "{self}/publish_file.py:{engine}/tk-multi-publish2/basic/nuke_publish_script.py"
```

Comment:
I don't think we should be inheriting from the base file plugin. This plugin doesn't actually publishing anything right?

Reply:
changed inheritance, see above